rust/ql/lib/codeql/rust/controlflow/internal/ControlFlowGraphImpl.qll

```diff
@@ -141,11 +138,10 @@ class LogicalAndTree extends PostOrderTree, LogicalAndExpr {
 
 class BlockExprTree extends StandardPostOrderTree, BlockExpr {
   override AstNode getChildNode(int i) {
-    result = super.getStmtList().getStatement(i)
```

Comment:
Why this change? I see inconsistent use of `instanceof SomeClass` and `extends ..., SomeClass` in this file. Wouldn't it be better to consistently use `instanceof` ?

Reply:
Better to consistently use `extends`.